test/hook.afterPackageInitialize.test.js

```diff
@@ -1,5 +1,6 @@
 /* global jest beforeEach describe it expect */
 
+const path = require('path');
```

Comment:
ESLint notes `Expected 1 empty line after require statement not followed by another require  import/newline-after-import`